extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -673,6 +673,17 @@ public void setSource(Source source) {
         @ConfigItem(defaultValue = "false")
         public boolean userInfoRequired;
 
+        /**
+         * Session age extension in minutes.
+         * The user session age property is set to the value of the ID token life-span by default and
+         * the user will be redirected to the OIDC provider to re-authenticate once the session has expired.
+         * If this property is set to a non-zero value then the expired ID token can be refreshed before
+         * the session has expired.
+         * This property will be ignored if the `token.refresh-expired` property has not been enabled.
+         */
+        @ConfigItem(defaultValue = "5M")
+        public Duration sessionAgeExtension = Duration.ofMinutes(5);
```

Comment:
Why minutes and not seconds as the code uses seconds?

Reply:
Hi @gastaldi, I was reasoning that it would be simpler for the users to set `30m` or say `45m` that start calculating `45*60` etc, it is harder to see for how long the session will be extended when looking at something like `1800s` etc.
The granularity is different here, compared say to `lifespan-grace` - there we just allow the users to add a few secs to the token lifespan...